ci/valgrind-check/src/pub_sub/bin/z_pub_sub.rs

```diff
@@ -38,7 +38,7 @@ async fn main() {
                 sample.key_expr().as_str(),
                 sample
                     .payload()
-                    .deserialize::<String>()
+                    .try_deserialize::<String>()
```

Comment:
Why changing the name? There is already quite a bit of material out there, not mentioning all bindings, that uses `deserialize`. I don't think this change is needed.

Reply:
everything has been changed in the last iteration with @DenisBiryukov91 and @kydos, see the coming PR. This one will be closed.